api/V1/Device.js

```diff
@@ -75,4 +78,40 @@ module.exports = function(app, baseUrl) {
         }
       });
   });
+
+  /**
+   * @api {get} /api/v1/devices Get list of devices
+   * @apiName GetDevices
+   * @apiGroup Devices
+   *
+   * @apiUse V1UserAuthorizationHeader
+   *
+   * @apiUse V1ResponseSuccess
+   * @apiSuccess {JSON} devices List of devices.
+   *
+   * @apiUse V1ResponseError
+   */
+  app.get(
+      apiUrl,
+      passport.authenticate(['jwt'], {session: false}),
+      async (request, response) => {
+          log.info(request.method + " Request: " + request.url);
+
+          // Check that the request was authenticated by a User.
+          if (!requestUtil.isFromAUser(request))
+            return responseUtil.notFromAUser(response);
+
+          var userGroupIds = await request.user.getGroupsIds();
+          var devices = await models.Device.findAndCount({
+              where: { GroupId: { "$in": userGroupIds } },
+              attributes: ["devicename", "id"],
+          });
```

Comment:
ordering?

Reply:
By device id or name?

Reply:
Name I guess? The user shouldn't need to know or care about device ids.